custom_components/solaredge_modbus_multi/number.py

```diff
@@ -107,33 +107,33 @@ def _handle_coordinator_update(self) -> None:
         self.async_write_ha_state()
 
 
-class StoredgeACChargeLimit(SolarEdgeNumberBase):
+class StorageACChargeLimit(SolarEdgeNumberBase):
     icon = "mdi:lightning-bolt"
 
     def __init__(self, platform, config_entry, coordinator):
         super().__init__(platform, config_entry, coordinator)
 
     @property
     def unique_id(self) -> str:
-        return f"{self._platform.uid_base}_storedge_ac_charge_limit"
+        return f"{self._platform.uid_base}_storage_ac_charge_limit"
 
     @property
     def name(self) -> str:
-        return "Storedge AC Charge Limit"
+        return "AC Charge Limit"
```

Comment:
All but 2 of the entities are prepended with "Storage" I think this should have the same. 
`        return "Storage AC Charge Limit"`
Also since you've renamed the control mode to Charge from Solar Power and Grid" maybe that should be reflected here?
`        return "Storage Grid Charge Power Limit"`
?

Reply:
Oh, sorry, missed that this was merged.